### ISSUE TITLE
Indico is sending reminders of a past meeting

### ISSUE THREAD

**Issue by pkoppenb** [indico/indico#4893]
**Describe the bug**
I large set of people have just (May 6) received the following e-mail
```
    Please note that the event "DPA WP3 Hackathon startup" will start on 27 Apr 2021, 10:00 (Europe/Zurich).

    You can access the full event here:
    https://indico.cern.ch/e/1032175

    Agenda for DPA WP3 Hackathon startup
    ------------------------------------
    There are no events scheduled.

    --
    Indico :: Email Notifier
    https://indico.cern.ch/e/1032175
```

**To Reproduce**
1. Take an indico meeting that has a reminder to one day before the meeting
2. Change its date to a past date
3. See reminder being sent

**Expected behavior**
Reminders should not be sent if the meeting is in the past (with some margin, Maybe one day).

**Additional context**
I realised that this meeting, which is just a copy of another meeting happening in another category was at a wrong date and reset it to the correct date. Then I got a reminder about a meeting last week.

**Comment by ThiefMaster** [comment]
I wouldn't necessarily call this a bug. The reminder is set to trigger x days before the event, and moving an event that has *not yet taken place* into the past does not seem like a common thing to do - and in fact for any other kind of re-scheduling you would probably want to keep the reminder set up.

That said, I don't see any benefit in sending such a reminder, so we could probably just delete it and show a message about it after changing the date in such a way...

**Comment by pkoppenb** [comment]
I agree. I tend to call "bug" every unexpected behaviour that can cause trouble (like here me having to reply to loads of e-mails of puzzled people). Thanks for the quick reply.

**Comment by ThiefMaster** [comment]
Interesting.. I just noticed this in the code. So cloning an event into the past should skip reminders, especially in your case:

```python
            # Skip anything that's would have been sent on a past date.
            # We ignore the time on purpose so cloning an event shortly before will
            # still trigger a reminder that's just a few hours overdue.
            if scheduled_dt.date() < now_utc().date():
                logger.info('Not cloning reminder %s which would trigger at %s', old_reminder, scheduled_dt)
                continue
```
